composer.json

```
@@ -22,7 +22,8 @@
     "require": {
         "php": "^7.4",
         "ext-json": "*",
-        "symfony/finder": "^4.3||^5.0.5"
+        "symfony/finder": "^4.3||^5.0.5",
+        "ext-zend-opcache": "*"
```

Since `config.sort-packages` is `true`, they should be sorted:
```suggestion
        "ext-zend-opcache": "*",
        "symfony/finder": "^4.3||^5.0.5"
```

---

No problems, fixed. Added this cause of using `opcache_get_status()` function without checking opcache ext.